examples/SharpBrick.PoweredUp.Examples/ExampleMoveHubInternalTachoMotorControl.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using SharpBrick.PoweredUp;
+
+namespace Example
+{
+    public class ExampleMoveHubInternalTachoMotorControl : BaseExample
```

Comment:
Maybe this could be renamed to ExampleMoveHubTachoMotorControl becaus i was running this and was not expecting that i have to plug in an external motor for that example and i got an exception because the `externalMotor` was null. As an alternative  the  null could be guarded.

Reply:
Oh sorry, that's my bad.  I've split the that into it's own example for now, so one example for internal motor and one when have an external motor connected.